Fixed wording in Rust 1.45 Blog Post (#654)
rust-lang/blog.rust-lang.org: posts/2020-07-16-Rust-1.45.0.md

```

On Rust 1.44.0, this happens to print "x: 0" on my machine. But it could
print anything, or do anything: this is undefined behavior. But the `unsafe`

> keyword is nowhere to be found within this block of code. This is what we call a

Reviewer: ```suggestion
keyword is not used within this block of code. This is what we call a
```
Re-reading, "nowhere to be found" seems a bit dramatic, what about this?

> keyword has not be specified in this code block. This is what we call a
> "soundness" bug, that is, it is a bug where the compiler does the wrong thing.

Reviewer 2: ```suggestion
keyword has not been specified in this code block. This is what we call a
"soundness" bug, where the compiler does the wrong thing.
```

minor suggestion re be vs been. using that is in the middle of a sentence is pretty weird as well, so maybe we can get two fixes in one.

Reviewer 3: This suggestion is good re: grammar, but using "specified" here sounds odd, because specification has a different meaning, technically. 

I can appreciate this sentence maybe sounding odd, but I think moving it in this direction makes it sound *more* odd, not less.

Author: Oops, yeah the "be" was a typo.  Ironic.

Author: What about

But the `unsafe` keyword is nowhere to be found within this block of code.

?

Reviewer: I prefer this latter formulation, fwiw.

Reviewer 3: @josephtlyons yeah that sounds good to me!
